CONTRIBUTING.md

```diff
@@ -14,12 +14,23 @@ To set up Feed Filter for local development, first install [Poetry][]:
 
 Go to the [Feed Filter repository][] and tap the **Fork** button at top-right. Then clone the source for your fork and add the upstream project as a Git remote:
 
-    git clone https://github.com/YOUR_USERNAME/feed-filter.git
-    cd feed-filter
-    git remote add upstream feed-filter.git
+    mkdir -p ~/projects/pelican-plugins ~/virtualenvs
```

Comment:
I would create `~/virtualenvs` right before using it, ie. in line 27. It seems clearer to me.

Reply:
Same thing in the detailed set-up.